system/settings/settings.xml

```diff
@@ -3188,7 +3181,7 @@
         </setting>
         <setting id="lookandfeel.enablerssfeeds" type="boolean" label="13305" help="36111">
           <level>0</level>
-          <default>true</default>
+          <default>false</default>
           <control type="toggle" />
```

Comment:
RSS changed to be disabled by default, this was from a conseus from a Team channel Slack/IRC conversion on Thur 16/06/2016.